src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import PeopleTable from './PeopleTable';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const fetchPeoples = async () => {
+      const response = await getPeople();
+
+      setPeople(response);
+    };
+
+    try {
+      fetchPeoples();
+    } catch (error) {
+      throw new Error(`${error}`);
```

Comment:
And you don't catch this error that you've thrown,  create a stateful value, like `errorText` and use conditional rendering to show it